pages/login.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { useSession, signIn, signOut } from 'next-auth/react';
+
+
+export default function Login() {
+    const {data: session} = useSession();
+
+    if (session) {
```

Comment:
Though not necessary, you could consider writing this using guard clauses, which is a standard code practice to avoid brief nesting of conditionals. This makes the code more readable with less indents